### DISCUSSION TITLE
Will 0.4 be released at all?

### DISCUSSION THREAD

**Discussion by ghost** [skyline-emu/skyline#1187] [Q&A] [answered]
Why is Pixelyon not releasing 0.4? He said it would come in 2 months back in 2019, 2 and a half years have passed and we never got 0.4.

**Comment by mateoalfaro** [comment]
Skyline keeps version 0.0.3 but it's still updated pretty much weekly

**Comment by PixelyIon** [comment, marked as answer]
We intentionally stopped versioned releases even though we've made plenty of progress, the reason for this is just that Skyline is currently developing and not in a final release state. Once we determine it is fit to do so we'll directly create 1.0.0 and release it on other channels like the Play Store and FDroid as well.

**Reply by ghost** [reply]
YES!
